packages/shopify-app-session-storage-dynamodb/loom.config.ts

```diff
@@ -0,0 +1,27 @@
+import {createPackage} from '@shopify/loom';
+import {buildLibrary} from '@shopify/loom-plugin-build-library';
+
+export default createPackage((pkg) => {
+  pkg.entry({root: './src/dynamodb.ts'});
+  pkg.use(
+    buildLibrary({
+      // Required. A browserslist string for specifying your target output.
+      // Use browser targets (e.g. `'defaults'`) if your package targets the browser,
+      // node targets (e.g. `'node 12.22'`) if your package targets node
+      // or both (e.g.`'defaults, node 12.22'`) if your package targets both
+      targets: 'node 14',
```

Comment:
Node 14 is technically not supported any more, should we bump this up to 16?

Reply:
Let's address that holistically across all the packages separately, as they all have this configuration.